### PR TITLE
Adds holochain_client_version variable to example package.json

### PR DESCRIPTION
**Pull request by matthme**
I haven't actually tested it yet but I'm assuming the example package.json.hbs has access to the `holochain_client_version` variable as well.

### CODE REVIEW

**Comment by c12i** [comment]
@matthme  unfortunately not, this is the only data passed to the example templates at  https://github.com/holochain/scaffolding/blob/develop/src/templates/example.rs#L24

I do think it makes sense to reference from the `versions.rs` rather than have them hard coded, it becomes very easy to miss updating these hard-coded values

```rs
 let data = ScaffoldExampleData {
        example: example.to_string(),
 };
```

**Comment by matthme** [comment]
Oh I see...yes I agree. It's really easy to miss them (as happened in this case).

File: templates/vanilla/example/ui/package.json.hbs [hunk 1]
```diff
@@ -9,7 +9,7 @@
       "package": "npm run build && cd dist && bestzip ../dist.zip *"
     },
     "dependencies": {
-      "@holochain/client": "^0.12.2",
+      "@holochain/client": "{{holochain_client_version}}",
```
Comment (c12i) [inline templates/vanilla/example/ui/package.json.hbs:12]: Prefer the hardcoded version for now

will create a PR to expose these values to the example templats

**Review by c12i — CHANGES_REQUESTED** [review]

**Comment by c12i** [comment]
Closing in favour of #218

**Pull request closed without merging**